src/features/smartSearch/components/types.ts

```diff
@@ -217,7 +223,7 @@ export interface SurveyOptionFilterConfig {
   survey: number;
   question: number;
   options: number[] | string[];
-  operator: CONDITION_OPERATOR;
+  operator: SURVEY_CONDITION_OP;
```

Comment:
This makes the type more correct (in relation to the API, which is the authority)!